spec/controllers/products_controller_spec.rb

```diff
@@ -0,0 +1,5 @@
+require 'rails_helper'
+
+RSpec.describe ProductsController, type: :controller do
+
```

Comment:
Let's make sure to write specs for the products controller as well - also try to write the tests first to follow TDD :) It gets some time to get used to, but it's good to get in the habit of it 🍺